model-builder/src/builder.rs

```diff
@@ -230,6 +230,7 @@ impl<'a> VehicleJourneyBuilder<'a> {
                 drop_off_type: 0u8,
                 datetime_estimated: false,
                 local_zone_id: None,
+                stop_time_precision: None,
```

Comment:
@datanel , @woshilapin 
How do we deal with changes on model-builder from now ?
There is always the Cargo.toml in version 0.3.0 :thinking:

Reply:
No idea. But excellent question. At some point, we could update it to the last version, make it support the entirety of the model, and then use it in our IT tests (instead of creating manually `Collections` and filling them with objects)... but I don't know when we'll have the time to do it.

Reply:
We will have to push the version 0.12 of transit_model (on crates.io), and after the version 0.3.1 of transit_model_builder (with transit_model 0.12 in dependency)